optional_plugins/resultsdb/avocado_resultsdb/__init__.py

```diff
@@ -142,6 +143,21 @@ def _status_map(status):
         return 'NEEDS_INSPECTION'
 
 
+class ResultsdbResult(Result):
+
+    """
+    ResultsDB render class
+    """
+
+    name = 'resultsdb'
+    description = 'Resultsdb result support'
+
+    def render(self, result, job):
+        if getattr(job.args, 'resultsdb_logs', None) is not None:
+            LOG_UI.info("JOB URL    : %s/%s" % (job.args.resultsdb_logs,
+                                                os.path.basename(job.logdir)))
+
```

Comment:
Cool. I will test this asap.

Reply:
Ok, it works as expected. Not sure `JOB URL` is the best label to put in our holy UI, but I don't have a better suggestion. As it will only show up when we're using `--resultsdb-logs`, it's not a big deal.
Another thing that I just realized is that we have to parse those URL's instead of just concatenating strings. But that will be required everywhere in this plugin, so it's not your patch's concern.